### PR TITLE
feat: Event Gateway handler [IA-227]

### PR DESCRIPTION
**Pull request by AlexWeinstein92**

### CODE REVIEW

File: organization/src/main/resources/application.conf [hunk 1]
```diff
@@ -32,6 +32,6 @@ akka.persistence {
 }
 datastax-java-driver.advanced.reconnect-on-init = true
 datastax-java-driver {
-  basic.contact-points = ["10.152.183.216:9042"]
+  basic.contact-points = ["scylla-db:9042"]
```
Comment (dag-yoppworks) [inline organization/src/main/resources/application.conf:35]: Glad this got sorted out

File: gateway/src/main/scala/com/improving/app/gateway/domain/common/util.scala [hunk 2]
```diff
@@ -21,4 +28,62 @@ object util {
       config.getInt(s"services.$serviceName.port")
     )
   }
+
+  def genEstablishTenantReqs(
```
Comment (dag-yoppworks) [inline gateway/src/main/scala/com/improving/app/gateway/domain/common/util.scala:32]: This really looks more like demo code, curious why it needed to move to the main part of the code base.

**Review by dag-yoppworks — APPROVED** [review]
No blockers here.

File: gateway/src/main/scala/com/improving/app/gateway/domain/common/util.scala (hunk 2 above)
Comment (AlexWeinstein92) [reply gateway/src/main/scala/com/improving/app/gateway/domain/common/util.scala:32]: Just refactoring to make it cleaner. It's where I decided to put it, but now we are moving all this to Gatling service, so it's no biggie

**Pull request merged**